### ISSUE TITLE
Decision on message files format

### ISSUE THREAD

**Issue by bram-atmire** [DSpace/dspace-angular#306]
Right now, we have a few translation files in the popular angular.json format. I understand that there are different tools to convert from and to this format. But the ngx-translate loader also supports other formats, including the .po format itself. So maybe we're better off switching to another format.

I'm personally not opinionated on this discussion. But before we invest more time in both extending the current language files, and getting files in for additional languages, we better formally settle on the format.

@helix84 @cjuergen what's you're opinion on this?

**Comment by cjuergen** [comment]
As I worked only with the initial translation and json and not with .po file before,  I'm not opinionated either.  

But from what I just read about the po format http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html it seems to have some advantages like being better to handle for non tech folks. 

There are a lot of well established tools like poedit  and online services to translate po files.

It seems good at handling missing/obsolete translations.
I like that it is human readable and the different types of (partly automated) comments.  
In json no comments are possible, and e.g. wanted to explain why the languages were in there original language here:
https://github.com/cjuergen/dspace-angular/blob/language-switch/resources/i18n/en.json#L277
which was not possible.

Using .po was proposed in #85 to ease the translation process.

**Comment by bram-atmire** [comment]
To summarize the decisions from the discussion in the linked pull request:
- for now, message keys are here to stay. We've tried the approach to use the English strings as the key, but we ran into to many serious blockers to run it through
- JSON5 is the format
- the decision to move to a hierarchical tree of messages, where you couldn't search on the message key anymore, was reverted. We now again have a linear list of message keys that makes it easier to navigate.

I'm very very much in favor to put together tooling to easily go from & to the JSON5 format, in order to support external tools & will be working on such initiatives.
